### PR TITLE
Set row level permissions with the 'give' command

### PR DESCRIPTION
**Pull request by tommydeboer**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/system tested
- [x] User documentation updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

**Comment by tommydeboer** [comment]
[Give RLS permissions in commander](https://trello.com/c/GQoZmAd5/534-give-rls-permissions-in-commander)

File: mcmd/commands/give.py [hunk 1]
```diff
@@ -76,54 +84,72 @@ def add_arguments(subparsers):
...
+                                                                            highlight(permission.value),
+                                                                            highlight(entity_id),
+                                                                            highlight(entity_type_id)))
+
+    security.grant_row_permission(principal_type, principal_name, entity_type_id, entity_id, permission)
 
 
 def _get_resource_type(args):
+    # The permission manager doesn't check if the resources actually exist so we need to do that ourselves
+
```
Comment (fdlk) [inline mcmd/commands/give.py:152]: Isn't that a bug that we should fix in molgenis?

File: mcmd/molgenis/security/permission_manager.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+def grant_permission(principal_type: PrincipalType,
+                     principal_name: str,
+                     resource_type: ResourceType,
+                     entity_type_id: str,
+                     permission: Permission):
+    """Grants permission on one (non-row) resource to one principal."""
+
+    data = {'radio-' + entity_type_id: permission.value}
```
Comment (fdlk) [inline mcmd/molgenis/security/permission_manager.py:20]: I'd expect all permission assignments to go through the permission API for servers that support it.

File: tests/integration/permission_client.py [hunk 3]
```diff
@@ -1,48 +1,62 @@
...
+
+
+def _get_permission(session, type_id, object_id, principal_type, principal) -> Optional[str]:
+    headers = _configure_headers(session)
     response = requests.get(urljoin(get_host()['url'],
-                                    'menu/admin/permissionmanager/{}/{}/{}'.format(resource_type, principal_type, id_)),
+                                    'api/permissions/{}/{}?q={}=={}'.format(type_id,
+                                                                            object_id,
+                                                                            principal_type,
+                                                                            principal)),
```
Comment (fdlk) [inline tests/integration/permission_client.py:39]: It's hard to decide what to trust and what not when writing these tests. This feels like it should be trusted code in the permissions api client, used in integration tests. Then in unit tests you could verify that the implementation calls the right endpoints.

File: mcmd/molgenis/api.py [hunk 4]
```diff
@@ -67,15 +67,20 @@ def import_by_url():
...
     return 'menu/admin/permissionmanager/update/'
 
 
 @endpoint
-def rls():
+def permission_manager_rls():
     return 'menu/admin/permissionmanager/update/entityclass/rls'
 
 
+@endpoint
```
Comment (fdlk) [inline mcmd/molgenis/api.py:79]: not completely sure how the @version annotation works but I'd expect one here

**Review by fdlk — CHANGES_REQUESTED** [review]
Can we take a look at this together?

**Review by fdlk — APPROVED** [review]

**Pull request merged**